spec/models/conversation_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe Conversation do
+  let(:alejandro) { create(:user, username: 'alejandro',
```

Comment:
You're generating a random username for your user on the factory. You can use that instead of hardcoding a new one here. Also, try to use `first_user,` `user,` or something similar instead of a `person name`. In this case, it could make sense what that variable represents, but it could confuse in so many cases.